dbms/src/DataStreams/copyData.cpp

```diff
@@ -51,6 +53,73 @@ void copyDataImpl(IBlockInputStream & from, IBlockOutputStream & to, TCancelCall
 
 inline void doNothing(const Block &) {}
 
+
+void copyDataImpl(BlockInputStreams & froms, BlockOutputStreams & tos)
+{
+    if (froms.size() == tos.size())
+    {
+        std::vector<std::thread> threads(froms.size());
```

Comment:
Need to use our ThreadPool (or ThreadFromGlobalPool) instead.
Because our threads are accountable.